KutokAccounting.Services/Stores/StoresService.cs

```diff
@@ -0,0 +1,67 @@
+using KutokAccounting.Services.Stores.Abstractions;
+using KutokAccounting.Services.Stores.Dtos;
+using KutokAccounting.Services.Stores.Extensions;
+using KutokAccounting.Services.Stores.Models;
+using KutokAccounting.Services.Vendors.Validators;
+
+namespace KutokAccounting.Services.Stores;
+
+public class StoresService : IStoresService
+{
+	private readonly IStoresRepository _repository;
+	private readonly StoreDtoValidator _storeDtoValidator;
+	public StoresService(IStoresRepository repository,
+		StoreDtoValidator storeDtoValidator)
+	{
+		_repository = repository;
+		_storeDtoValidator = storeDtoValidator;
+	}
+	
+	public async ValueTask CreateStoreAsync(StoreDto storeDto, CancellationToken ct)
+	{
+		ct.ThrowIfCancellationRequested();
+
+		var storeModel = storeDto.FromDto();
+		await _repository.CreateStoreAsync(storeModel, ct);
+	}
+
+	public async ValueTask<List<StoreDto>> GetStoresPageAsync(Page page, CancellationToken ct, SearchParameters? searchParameters = null)
+	{
+		ct.ThrowIfCancellationRequested();
+		
+		var stores = await _repository.GetFilteredPageOfStoresAsync(page, ct, searchParameters);
+		return stores.Select(x => x.ToDto()).ToList();
```

Comment:
Чи тут справді потрібна конвертація в дто?
```suggestion
		return stores;
```